lib/block_view_flush.go

```diff
@@ -1438,7 +1410,7 @@ func (bav *UtxoView) _flushPostAssociationEntriesToDbWithTxn(txn *badger.Txn, bl
 		} else {
 			// If the AssociationEntry has isDeleted=false then we
 			// put the corresponding mappings for it into the db.
-			if err := DBPutPostAssociationWithTxn(txn, bav.Snapshot, &associationEntry, blockHeight); err != nil {
+			if err := DBPutPostAssociationWithTxn(txn, bav.Snapshot, &associationEntry, blockHeight, bav.EventManager); err != nil {
```

Comment:
Just a note: we'll need to update the flush logic to support the event manager for all the new PoS entries.